crates/egui_plot/src/items/mod.rs

```diff
@@ -1234,12 +1234,13 @@ impl PlotItem for PlotImage {
             Rect::from_two_pos(left_top_screen, right_bottom_screen)
         };
         let screen_rotation = -*rotation as f32;
-        RawImage::new((*texture_id, image_screen_rect.size()))
-            .bg_fill(*bg_fill)
-            .tint(*tint)
-            .uv(*uv)
-            .rotate(screen_rotation, Vec2::splat(0.5))
-            .paint_at(ui, image_screen_rect);
+        ui.add(
+            Image::from_texture((*texture_id, image_screen_rect.size()))
+                .bg_fill(*bg_fill)
+                .tint(*tint)
+                .uv(*uv)
+                .rotate(screen_rotation, Vec2::splat(0.5)),
+        );
```

Comment:
no, calling `ui.add` here won't work well at all. That will put the image just anywhere. Check out the `Plot` demo in `egui_demo_app`: 
![image](https://github.com/emilk/egui/assets/1148717/59f15677-192a-4908-bd72-0878b34a6d36)

You should still use `Image::paint_at` here.

Btw, that plot demo could also use the egui icon image instead of `ColorImage::example`.